client/src/App.vue

```diff
@@ -2,14 +2,16 @@
   <div id="app" class="font-sans">
     <div id="nav"><SiteHeader /></div>
```

Comment:
I know you didn't add this in this pull, but I think we can delete the `<div id="nav">` here, right?
```suggestion
    <SiteHeader />
```